charts/aws-ebs-csi-driver/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.22.0
 name: aws-ebs-csi-driver
 description: A Helm chart for AWS EBS CSI Driver
-version: 2.22.0
+version: 2.22.1
```

Comment:
Thanks for adding this : ) 

We typically have a maintainer who performs releases make these changes so let's remove it.